Doc/library/math.rst

```diff
@@ -790,29 +839,5 @@ Constants
       It is now always available.
 
 
-.. impl-detail::
-
-   The :mod:`math` module consists mostly of thin wrappers around the platform C
-   math library functions.  Behavior in exceptional cases follows Annex F of
-   the C99 standard where appropriate.  The current implementation will raise
-   :exc:`ValueError` for invalid operations like ``sqrt(-1.0)`` or ``log(0.0)``
-   (where C99 Annex F recommends signaling invalid operation or divide-by-zero),
-   and :exc:`OverflowError` for results that overflow (for example,
-   ``exp(1000.0)``).  A NaN will not be returned from any of the functions
-   above unless one or more of the input arguments was a NaN; in that case,
-   most functions will return a NaN, but (again following C99 Annex F) there
-   are some exceptions to this rule, for example ``pow(float('nan'), 0.0)`` or
-   ``hypot(float('nan'), float('inf'))``.
-
-   Note that Python makes no effort to distinguish signaling NaNs from
-   quiet NaNs, and behavior for signaling NaNs remains unspecified.
-   Typical behavior is to treat all NaNs as though they were quiet.
-
-
-.. seealso::
-
-   Module :mod:`cmath`
-      Complex number versions of many of these functions.
-
```

Comment:
One last nitpick.  Lets not move this footer.

Reply:
I had two things in mind when moving this block:
* I find it not easily discoverable, I wasn't expecting another general paragraph *after* the API listing
* In the page layout, it effectively appears in the `Constants` section, which I find confusing.

I propose to put it back at the end as you asked, and to create a new section named `Implementation detail`. That would fix both of my concerns.

Reply:
> I find it not easily discoverable

It a good thing.

> In the page layout, it effectively appears in the Constants section, which I find confusing.

I think that it's visually distinct from the sphinx docs of constants.  No need for extra section.

Reply:
Alright, the block is back to the end of the file.